tools/expotools/src/dynamic-macros/IosMacrosGenerator.ts

```diff
@@ -205,6 +205,30 @@ async function renderExpoKitPodfileAsync(
   });
 }
 
+async function generateDefinesFile(): Promise<void> {
+  const oldestSDKVersionString = await ProjectVersions.getOldestSDKVersionAsync("ios");
```

Comment:
```suggestion
  const oldestSDKVersionString = await ProjectVersions.getOldestSDKVersionAsync('ios');
```

Reply:
IMO, we should ensure this coding style via some linting process